src/config/dynamic-theme-fixes.config

```diff
@@ -7472,6 +7472,44 @@ div.sumo-nav--logo
 
 ================================
 
+sverigesradio.se
+
+INVERT
+.cross
+.default
+.details-button__icon
+.external-link-with-icon__icon
+.gallery-button svg g
+.info-teaser-container__title h2
+.link-icon .icon
+.local-weather-item__wind-icon
+.logo
+.menu-icon.icon
+.play-icon--white
+.program-footer__link-item:nth-of-type(2) svg
+.progress.queue-progress .bar
+.support-info__icon svg
+input[type="range"]::-webkit-slider-thumb
+input[type="range"]::-moz-range-thumb
+input[type="range"]::-webkit-slider-runnable-track
+input[type="range"]::-moz-range-track
+button.reset
+
+CSS
+.local-weather-item__weather-icon {
+  filter: invert(1) hue-rotate(180deg) !important;
+}
+
+.live-marker .dot, .live-label::before {
+  background-color: #fff !important;
```

Comment:
4 spaces and I think the `#fff` can be changed to `var(--darkreader-neutral-text)`